managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/OpReadEntry.java

```diff
@@ -34,6 +36,9 @@
 
 class OpReadEntry implements ReadEntriesCallback {
 
+    private static AtomicInteger createCount = null;
```

Comment:
we should do this with PowerMock

please do not add such instrumentation only for testing

Reply:
Done. PTAL again. BTW, I found Mockito can do that as well.